thegreatcain/cocam-automation
# 1 implement signin tests

This PR implements tests for the signin page forms.

## How to Test
1.  Navigate to `<cocam repo>/cypress/integration/signin_page/` and verify a new cocam_signin_spec.js file exists
2.  Launch the cypress test runner
3. Select the cocam_signin_spec.js file and watch the runner execute the tests
4. Observe there should be 6 tests included that execute and (hopefully) pass.
